src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  let sexPerson = 'wife';
+
+  if (sex === 'f') {
+    sexPerson = 'husband';
+  }
```

Comment:
replace to const and ternary operator
```suggestion
  const sexPerson = sex === 'f' ? 'husband' : 'wife';
```